packages/schematics/angular/application/files/module-files/src/app/app.component.spec.ts.template

```diff
@@ -1,12 +1,12 @@
 import { TestBed } from '@angular/core/testing';<% if (routing) { %>
-import { RouterTestingModule } from '@angular/router/testing';<% } %>
+import { RouterModule } from '@angular/router';<% } %>
 import { AppComponent } from './app.component';
 
 describe('AppComponent', () => {
   beforeEach(async () => {
     await TestBed.configureTestingModule({<% if (routing) { %>
       imports: [
-        RouterTestingModule
+        RouterModule.forRoot([])
```

Comment:
Wouldn't `RouterModule` be sufficient in this case?

Reply:
Maybe just `RouterOutlet` ?

Reply:
Yes just using RouterModule should be sufficient since we are not registering any routes, but I found [this comment](https://github.com/angular/angular/issues/54853#issuecomment-1997551229) when looking at related issues, which is why I updated it to use `RouterModule.forRoot` with an empty list of routes. Please let me know if you think it is unnecessary and can be removed.

Reply:
@atscott, what do you suggest here?

Reply:
RouterOutlet is the minimal thing to have in the imports here. If you aren’t trying to import anything more than is necessary, that’s the thing to do.

If you want developers to not have to learn about what parts they’re using and need to import, then RouterModule.forRoot([]) makes it work for cases where RouterLink is added to the component template (RouterModule alone would not).

Reply:
Thanks for your response @atscott. @alan-agius4, @eneajaho From an ease-of-use standpoint I feel it may be better to use `RouterModule.forRoot([])`, but I can make the change to use `RouterOutlet`, if you prefer that instead.